bedrock/firefox/redirects.py

```diff
@@ -241,4 +241,7 @@ def firefox_mobile_faq(request, *args, **kwargs):
 
     # bug 960543
     redirect(r'^firefox/(?P<vers>[23])\.0/eula', '/legal/eula/firefox-{vers}/'),
+
+    # bug 1150713
+    redirect(r'^firefox/sms(?:/.*)?$', '/firefox/products/'),
```

Comment:
Maybe. I based the regex off one above which included the trailing `$`. If a new page with the path `sms-foo` is added the updated redirect will stomp on it.